packages/components/pagination/src/Pagination.tsx

```diff
@@ -85,6 +85,14 @@ export interface PaginationProps extends CommonProps {
    * @default { labelNext: "Next", labelPrevious: "Previous", ariaLabelNext: "To next page", ariaLabelPrevious: "To previous page" }
    */
   navigationButtonsProps?: NavigationButtonsProps;
+  /**
+   * Labels for the text when total items are shown
```

Comment:
Maybe we can add an example in the description so it's easier to understand what part of the text is. Also mentioning that this is the postfix and not the full text with both variables is expected here:
```suggestion
   * Postfix Labels for the current number of items when total items are shown e.g., `of ${totaltems} items`
```